service/jargon/defaults.yaml

```diff
@@ -1,91 +1,291 @@
+#Default jargon - comment out as necessary. 
+
+#Note that some of these are experimental abbreviations which don't have any real purpose and are simply there to play with
+ 
+#shortcuts
+ 
 1: I
 2: II
-aar: Ancillary Armor Repairer
+lrg: Large
+hvy: Heavy
+med: Medium
```

Comment:
`med: Medium` is ineffective since any search for "med" will already pull up every "medium" thing.